src/main/java/org/folio/oaipmh/helpers/RepositoryConfigurationUtil.java

```diff
@@ -112,4 +114,11 @@ public static boolean getBooleanProperty(Map<String, String> okapiHeaders, Strin
     return parseBoolean(defaultValue);
   }
 
+  public static boolean isDeletedRecordsEnabled(Request request, String name){
+    String tenant = TenantTool.tenantId(request.getOkapiHeaders());
+    String propertyName = getProperty(tenant, name);
+
+    return (DeletedRecordType.fromValue(propertyName)).equals(DeletedRecordType.PERSISTENT)
```

Comment:
So, if there is no enum instance for 'propertyName' then exception will be throwed. Please wrap it with try catch and wihin the catch block  set up the default value for setting. You can watch the default value within config file into resource dirrectory